addon/addon.go

```diff
@@ -166,3 +169,41 @@ func NewAddonManager(kubeConfig *rest.Config, agentImage string, agentInstallAll
 
 	return addonMgr, err
 }
+
+func GetMchImage(kubeConfig *rest.Config) string {
+	labelSelector := &metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"ocm-configmap-type": "image-manifest",
+		},
+	}
+
+	mchSelector, err := appsubutils.ConvertLabels(labelSelector)
+	if err != nil {
+		klog.Error("Failed to convert label", err)
+		return ""
+	}
+
+	mchImageCMList := &corev1.ConfigMapList{}
+
+	kubeClient, err := client.New(kubeConfig, client.Options{})
+	if err != nil {
+		klog.Errorf("Unable to kube client: %v", err)
+		return ""
+	}
+
+	err = kubeClient.List(context.TODO(), mchImageCMList, &client.ListOptions{LabelSelector: mchSelector})
+	if err != nil {
+		klog.Error(err, "Failed to get configmap for MCH images")
+		return ""
+	}
+
+	if len(mchImageCMList.Items) > 0 {
+		data := mchImageCMList.Items[0].Data
```

Comment:
there is an upgrade case need to be considered here.
eg. there will be 2 configmaps  after upgrade from 2.4 to 2.5, mch-image-manifest-2.4.3 and mch-image-manifest-2.5.0. they have the same label `ocm-configmap-type=image-manifest `, but have another different labels `ocm-release-version=2.4.3` and `ocm-release-version=2.5.0`.
if you decide to get image tag from the configmap, you need to maintain the ACM release version too. 
yes we can do it in midstream, but I don't think this is a proper solution for upstream. @xiangjingli